release/jo/jo.wbl-cyrl-tj.wakhi_cyrillic_minimal/jo.wbl-cyrl-tj.wakhi_cyrillic_minimal.kpj

```diff
@@ -2,7 +2,7 @@
 <KeymanDeveloperProject>
   <Options>
     <BuildPath>$PROJECTPATH\build</BuildPath>
-    <CompilerWarningsAsErrors>True</CompilerWarningsAsErrors>
+    <CompilerWarningsAsErrors>False</CompilerWarningsAsErrors>

```

Comment:
That latest lexical model compiler beta contains keymanapp/keyman#4563 which will treat the BCP-47 tag as "info" instead of "warn". Then you can keep this `True`. 

In package.json at the root of this repo: edit the `@keymanapp` dependencies:
```
  "dependencies": {
    "@keymanapp/lexical-model-compiler": "^14.0.253-beta",
    "@keymanapp/models-types": "^14.0.253-beta",
    "@types/node": "^10.17.27",
```

Then run `npm install`.
You'll want to attach the package.json and package-lock.json files to this PR too.

Reply:
Though it's probably "purer" you do that on a separate PR first. Then this PR and @MakaraSok 's PR can merge the update

Reply:
There's gonna be an imminent update the the lexical model compiler for this repo.

See #124

This PR will probably need to rebase after that.

Reply:
I'll close this PR and submit another (with the WarningsAsErrors flag set True) once the new compiler is in place.